src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="imac"
        class="card__image"
      >
      
      <div class="card__name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </div>
```
fix all such cases
<img width="947" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/3e58e2f4-7841-4365-a02c-fa46c0409ce3">